strassengezwitscher/strassengezwitscher/settings/default.py

```diff
@@ -117,6 +118,15 @@
     'auth.user': lambda u: '/intern/users/%s/' % u.id,
 }
 
+######################################################################################################
+#                                           Recaptcha settings
+######################################################################################################
+
+# This is key for development purposes only
+RECAPTCHA_SECRET_KEY = ['6LeIxAcTAAAAAGG-vFI1TnRWxMZNFuojJ4WifJWe']
```

Comment:
Ich finde `default.py` ist schon die richtige Stelle dafür. Bloß der Kommentar triffts nicht so ganz. Im Produktiveinsatz werden wir den Wert in `sensitive_settings.py` überschreiben. Sollten wir das aber vergessen und der Wert wurde nur in `dev.py` gesetzt, dann ist der Wert bei Ausführung nicht vorhanden, das führt zu Fehlern. Dann würde die Lauffähigkeit unseres Codes von der optionalen Datei `sensitive_settings.py` abhängen.
Von daher ist mir ein Default-Wert für alle Umgebungen ganz recht, den wir dann aber entsprechend überschreiben.